src/CharacterManager.hpp

```diff
@@ -54,6 +54,8 @@ namespace SokuLib
 		float zRotation;
 	};
 
+    //! @brief An Object is any entity that exists within a battle: characters, bullets, visual effects, etc.
```

Comment:
You could also say that it corresponds to an AnimationObject in the v2 namespace